atlas/AtlasMl/atlasml/ml/MLPipelines/PipelineWorkflows.py

```
@@ -0,0 +1,173 @@
+import uuid
+
+import numpy as np
+import uuid
```

Ruff (0.11.9)

Redefinition of unused `uuid` from line 1

Remove definition: `uuid`

(F811)